src/displayapp/screens/Gallery.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/screens/FileView.h"
+#include "displayapp/DisplayApp.h"
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Screens {
+      class Gallery : public Screen {
+      public:
+        static constexpr const char* directory = "/gallery/";
+
+        Gallery(DisplayApp* app, Pinetime::Controllers::FS& filesystem);
+        ~Gallery() override;
+        bool OnTouchEvent(Pinetime::Applications::TouchEvents event) override;
+
+      private:
+        int string_ends_with(const char* str, const char* suffix);
```

Comment:
InfiniTime uses CamelCase for function names, so `StringEndsWith` as well as `ListDir` and `Open`.

Reply:
Fixed with e6b5a159